React/Views/RCTSlider.m

```diff
@@ -32,4 +32,18 @@ - (void)setMaximumValue:(float)maximumValue
   super.value = _unclippedValue;
 }
 
+- (void)setThumbImage:(NSDictionary *) thumbImage
+{
+  UIImage *image = [UIImage imageNamed:thumbImage[@"source"]];
+
+  CGSize newSize = CGSizeMake([thumbImage[@"width"] floatValue], [thumbImage[@"height"] floatValue]);
```

Comment:
Resizing images on the main thread like this is generally a bad idea as it can cause frame skips.